rviz_common/src/rviz_common/visualization_manager.cpp

```diff
@@ -436,7 +438,7 @@ void VisualizationManager::onUpdate()
 
   frame_count_++;
 
-  if (render_requested_ || wall_dt > 0.01) {
+  if (render_requested_ || wall_dt > 0) {
```

Comment:
I don't think this is right, in the original code, it was comparing floating point seconds to `0.01`, now we're using nanoseconds. So this is definitely a bug, but I think the right solution is to compare it to `10ms` rather than zero.

Reply:
Original: https://github.com/ros-visualization/rviz/blob/f9b357fdc0bfea5361e27b506059e62eab1f861b/src/rviz/visualization_manager.cpp#L336

Reply:
> So this is definitely a bug, but I think the right solution is to compare it to 10ms rather than zero.

@wjwwood that makes a lot of sense. So the correct comparison would be `wall_dt > 1e+7`?

Reply:
I believe `1e7` will be floating point, I would use `std::chrono::milliseconds(10)` instead.

Reply:
Or if you wanted to be really specific, `wall_dt > std::chrono::duration_cast<decltype(wall_dt)>(std::chrono::millisecond(10))`, but I think the `operator>` should do this for chrono types already.